calculator/src/App.js

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import ButtonPad from "./components/buttons";
+import "./App.css";
+import { render } from "react-dom";
+import "./master.css";
+import { evaluate } from "mathjs";
+
+class App extends React.Component {
+  state = {
+    currentValue: "",
+    prevValue: "",
+    operation: "",
+    answer: ""
+  };
+
+  handleChange = e => {
+    let input = e.target.value;
+    if (this.state.operation !== "") {
+      this.setState(prevState => ({
+        prevValue: prevState.currentValue,
+        currentValue: input
+      }));
+      // } else if (this.state.currentValue.length >= 4) {
+      //   let comma = this.state.currentValue;
+      //   this.setState({
+      //     currentValue: comma.toLocaleString()
+      //   });
+    } else {
+      this.setState(prevState => ({
+        currentValue: prevState.currentValue + input
+      }));
+    }
+  };
+
+  handleEqual = () => {
+    let evaluation = eval(
```

Comment:
You need to use try - catch , otherwise if something doesn't go as planned - the app breaks.